File: Selection Sort.cpp
int select(int arr[], int i)
{
    return arr[i];
} // code here such that selecionSort() sorts arr[]



void selectionSort(int arr[], int n)
{
   for(int i=0; i<n-1; i++){
      for(int j=i+1; j<n; j++){
          if(arr[j]<arr[i]){
              int temp = arr[j];
              arr[j] = arr[i];
              arr[i] = temp;
          }
      }
   }
  //code here
}

File: Selection sort.py
class Solution: 
    def select(self, arr, i):
        minimum = i
        for j in range(i+1, n):
            if arr[j] < arr[minimum]:
                minimum = j
        return minimum
    def selectionSort(self, arr,n):
        for i in range(n-1):
            minimum = self.select(arr, i)
            arr[minimum], arr[i] = arr[i], arr[minimum]
